tfe/resource_tfe_organization.go

```diff
@@ -0,0 +1,156 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceTFEOrganization() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTFEOrganizationCreate,
+		Read:   resourceTFEOrganizationRead,
+		Update: resourceTFEOrganizationUpdate,
+		Delete: resourceTFEOrganizationDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"email": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"session_timeout": &schema.Schema{
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  20160,
+			},
+
+			"session_remember": &schema.Schema{
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  20160,
+			},
+
+			"collaborator_auth_policy": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: func(v interface{}, k string) ([]string, []error) {
+					switch v.(string) {
+					case "password", "two_factor_mandatory":
+						// These are the two valid options, so we return without errors.
+						return nil, nil
+					default:
+						return nil, []error{fmt.Errorf(
+							"%q must be either 'password' or 'two_factor_mandatory'", k),
+						}
+					}
```

Comment:
_minor_ we can replace this with `validation.StringInSlice([]string{ "password", "two_factor_mandatory" }, false)`